src/ansys/fluent/core/quantity/dimensions.py

```diff
@@ -0,0 +1,123 @@
+import ansys.fluent.core.quantity as q
+
+
+class Dimensions(object):
+    """Initialize a Dimensions object using a dimensions list or a unit string.
+
+    Parameters
+    ----------
+    units: str
+        Unit string of quantity.
+    dimensions: list
+        List of dimensions.
+
+    Returns
+    -------
+    Dimensions instance.
+    """
+
+    def __init__(
+        self, units: str = None, dimensions: list = None, unit_sys: str = None
+    ):
+        self._units_table = q.UnitsTable()
+        unit_sys = unit_sys or "SI"
+        units = units or " "
+
+        if units:
```

Comment:
This is always `True` given the line above.

Reply:
Fixed using `if units is not None:`

https://github.com/ansys/pyfluent/blob/911b052f1e7162a45f2bc72965029fcf5045470b/src/ansys/fluent/core/quantity/dimensions.py#L27-L32